debezium-core/src/main/java/io/debezium/relational/history/KafkaDatabaseHistory.java

```diff
@@ -562,8 +532,7 @@ private short getDefaultTopicReplicationFactor(AdminClient admin) throws Excepti
             if (defaultReplicationFactorValue != null) {
                 return Short.parseShort(defaultReplicationFactorValue);
             }
-        }
-        catch (ExecutionException ex) {
+        } catch (ExecutionException ex) {
```

Comment:
This looks like a reformat change that was unnecessary, no?

Reply:
Yeah these are not required I will fix it